README.md

```diff
@@ -78,19 +78,20 @@ Extend `Butler\Health\Check` and add it to `butler.health.checks`, done.
 
 ## Custom about information
 
-You can push additional "about" information via [laravels `AboutCommand`](https://laravel.com/docs/9.x/packages#about-artisan-command) class.
+You can push additional "about" information.
 
 ```php
-AboutCommand::add('Environment', fn () => [
-    'Operating System' => php_uname('s'),
-]);
+Repository::add('environment', ['operating_system' => php_uname('s')]);
```

Comment:
```suggestion
Butler\Health\Repository::add('environment', ['operating_system' => php_uname('s')]);
```

?

Just `Repository` is a somewhat generic name. `AboutRepository`? Or, as suggested, using the full namespace to make it clearer in the documentation, at least?